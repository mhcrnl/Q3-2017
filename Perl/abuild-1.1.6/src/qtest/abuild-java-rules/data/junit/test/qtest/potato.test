#!/usr/bin/env perl
require 5.008;
BEGIN { $^W = 1; }
use strict;

require TestDriver;

my $td = new TestDriver('potato');

$td->runtest("potato",
	     {$td->COMMAND => "test_potato"},
	     {$td->FILE => "potato.out", $td->EXIT_STATUS => 0},
	     $td->NORMALIZE_NEWLINES);

$td->runtest("fail",
	     {$td->COMMAND => "test_potato fail"},
	     {$td->FILE => "potato-fail.out", $td->EXIT_STATUS => 0},
	     $td->NORMALIZE_NEWLINES);

$td->report(2);
