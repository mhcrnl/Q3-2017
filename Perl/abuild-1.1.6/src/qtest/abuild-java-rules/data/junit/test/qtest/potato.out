raw potato
baked potato
In junit setup
In junit cleanup with fried potato and roasted potato
In junit setup
In junit cleanup with fried potato and roasted potato
In junit setup
In junit cleanup with fried potato and roasted potato
JUnit tests passed
